src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,12 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
+
+	private static final String SKRIPT_PREFIX = "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
+	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix");
```

Comment:
With Tud's [requested change](https://github.com/SkriptLang/Skript/pull/5437#discussion_r1104235017) this would be 
```suggestion
	private static final Message SKRIPT_PREFIX_MESSAGE = new Message("skript.prefix").getValueOrDefault(SKRIPT_PREFIX);
```

Reply:
Hmm we need to store it as an actual Message though to handle any runtime lang changes

Reply:
Actually yea my example is wrong, SKRIPT_PREFIX is string not Message

Reply:
Alright this should now be resolved didn't see this was made at that time and since has been fixed